cohortextractor/process_covariate_definitions.py

```diff
@@ -334,46 +334,45 @@ def _type_from_return_value(self, returning):
                 "'nuts1_region_name'"
             )
         mapping = {
+            "administrative_category": "str",
+            "admission_method": "str",
+            "admission_treatment_function_code": "str",
             "binary_flag": "bool",
+            "case_category": "str",
+            "category": "str",
+            "code": "str",
             "date": "date",
             "date_admitted": "date",
+            "date_arrived": "date",
             "date_discharged": "date",
             "date_of_death": "date",
+            "days_in_critical_care": "str",
+            "discharge_destination": "str",
+            "duration_of_elective_wait": "str",
+            "group_16": "str",
+            "group_6": "str",
```

Comment:
this ordering isn't enforced, we could order numerically here rather than lexicographically